src/blocked.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Redis Ltd. 2023 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#include "redisraft.h"
+
+#include <string.h>
+
+/* A list of all current blocked commands */
+
+BlockedCommand *addBlockedCommand(raft_index_t idx, unsigned long long session, const char *data, size_t data_len, RaftReq *req, RedisModuleCallReply *reply)
```

Comment:
```suggestion
BlockedCommand *addBlockedCommand(raft_index_t idx, unsigned long long session, const char *data, size_t data_len, RaftReq *req)
```

Looks like you don't use reply here

Reply:
use it now